spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -2,8 +2,25 @@
 
 RSpec.describe InvenioRdmRecordConverter do
   let(:user) { FactoryGirl.create(:user) }
-  let(:generic_file) { make_generic_file(user, doi: ["doi:123/ABC"]) }
-  let(:json) { "{\"pids\":{\"doi\":{\"identifier\":\"#{generic_file.doi.shift}\",\"provider\":\"datacite\",\"client\":\"digitalhub\"}},\"provenance\":\"#{user.username}\"}" }
+  let(:generic_file) { make_generic_file(user, doi: ["doi:123/ABC"], resource_type: ["Account Books"]) }
+  let(:json) do
+    {
+      "pids": {
+        "doi": {
+          "identifier": "#{generic_file.doi.shift}",
+          "provider":"datacite",
+          "client":"digitalhub"
+        }
+      },
+      "provenance":"#{user.username}",
+      "metadata": {
+        "resource_type": {
+          "type": "Books",
```

Comment:
There are types that can be assigned for a file in DH that are not being used at all. For example, textbooks are supported, but not used. For now I think this is a good enough test, but at some point it would probably be good to either sample an array of types that we know we need to support OR modify the factory to create more generic files to convert. I think the latter idea is the better one, but will take more effort. Testing stuff always causes me problems it seems.

Reply:
Thanks for the explanation. I think this is good enough for now, but let's make a note to improve tests down the road if we are able. And thanks for redefining `json` here, I don't know what I was thinking keeping it one-line.